data/README.md

```diff
@@ -0,0 +1,3 @@
+# Updating [git-platinum](https://github.com/radicle-dev/git-platinum)
+Just `cd` into the repo root, then run `scripts/update-git-platinum.sh`.
+This will update the tarball and commit the change.
```

Comment:
We need to change this instruction now to tell the person to commit the change :) I'd also suggest a template:

```suggestion
This will update the tarball and it's necessary to commit the change. We provide a template below so that we can easily identify changes to `git-platinum`. Please fill in the details that follow a comment (`#`):


    data/git-platinum: # short reason for updating

    # provide a longer reason for making changes to git-platinum
    # as well as what has changed.

```